src/nodes/fmod_event_emitter_2d.cpp

```diff
@@ -0,0 +1,194 @@
+//
+// Created by bitbrain on 2022-08-14.
+//
+
+#include "fmod_event_emitter_2d.h"
+#include "../godot_fmod.h"
```

Comment:
You should not use relative path for includes, you should rather use imports with ˋ<>`. Include path should be setup in scons (and cmake for édition with clion).